backend/src/config/theme.rs

```diff
@@ -166,7 +166,7 @@ impl ThemeConfig {
         let happy = Hsl::from(self.color.happy);
         add!("--happy-hue" => format_args!("{:.2}", happy.h));
         add!("--happy-sat" => format_args!("{:.2}%", happy.s * 100.0));
-        add!("--happy-lightness" => format_args!("{:.2}%", happy.l * 100.0));
+        add!("--happy-lightness" => format_args!("{:.2}%", happy.l * 77.5));
```

Comment:
For one, I wouldn't change the happy color in this spot, BUT I think that doesn't matter because this problem can be fixed in a better way I think.

I just checked our happy color (`#27ae60`) with [this tool](https://webaim.org/resources/contrastchecker/) for black and white text. Black text has a way better contrast according to that. So why do we use white? We already have a way to choose between black and white depending on which has the larger contrast. That's the `var(--happy-color-bw-contrast)` in CSS. That is set in this file, a few lines down. 

So I think that the `bw_contrast` function in `color.rs` is simply incorrect or not useful. So what you could do is improve that. Check what "formulas" well known tools for accessibility contrast checking use and then use the same logic in `bw_contrast`.

Reply:
Looking at the redesign, this will also be redundant once that gets applied. So I think this PR can be closed?

Reply:
Of course I could change the `bw_contrast` function anyway.

Reply:
> Of course I could change the `bw_contrast` function anyway.

I think that is useful regardless. Yes, the redesign might change the default colors, but people can still configure them and we need an automatic "black or white depending on what has better contrast" function regardless. So the one we have should work well.